ui/src/components/select/QasSelect.vue

```diff
@@ -163,7 +173,7 @@ export default {
 
   methods: {
     setFuse () {
```

Comment:
Doug, no `created` tá chamando esse método e na computada `hasFuse` tem uma condição `this.useSearch || this.options.length >= autoFuseQuantity`, dito isso, vem minha dúvida:

Será que a condição `this.options.length >= autoFuseQuantity` não vai permitir que o Fuse seja instanciado para um `QasSelect` lazy loading?